exercises/03_count_byte/03_count_byte_test.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import "testing"
+
+func TestCountByte(t *testing.T) {
+	for _, tc := range []struct {
+		name  string
+		str   string
+		sbyte byte
+		want  int
+	}{
+		{"hello", "hello", 0x6c, 2},
+		{"empty", "", 0x6c, 0},
+		{"Latvian letters", "abčdē", 0xc4, 2},
```

Comment:
In most cases, instead of explicit hex values, please use runes converted to `byte`, e.g. `byte('l')`.

Obviously, you need one or several tests where `sbyte` is a byte that doesn't have a corresponding printable character.